README.md

```diff
@@ -219,6 +220,19 @@ await client.logon();
 await client.logoff();
 ```
 
+## onSessionExpiration callback
+The onSessionExpiration is callback with the sdk client as parameter, it is called when the ACC session is expired.
+The callback return no thing. We can use the callback to reconnect the sdk client.
+
+```js
+const connectionParameters = sdk.ConnectionParameters.ofUserAndPassword(
+                                url, "admin", "admin",
+                                {onSessionExpiration: async (client)=>{
+                                        await client.logon();
```

Comment:
What if logon returns an expired session again? How do we prevent infinite loops?